inc/itilsolution.class.php

```diff
@@ -83,7 +83,7 @@ public static function pdfForItem(PluginPdfSimplePDF $pdf, CommonDBTM $item)
                 } else {
                     $text = $textapprove = '';
                 }
-                if (isset($row['date_approval']) || isset($row['users_id_approval'])) {
+                if (isset($row['date_approval']) || !empty($row['users_id_approval'])) {
```

Comment:
Should not be 
```suggestion
                if (isset($row['date_approval']) && !empty($row['users_id_approval'])) {
```

Reply:
ping @Rom1-B

Reply:
Both fields are not mandatory, but at least 1, so OR is suitable